core/Cargo.toml

```diff
@@ -9,4 +9,7 @@ holochain_agent = { path = "../agent" }
 chrono = "0.4"
 wasmi = { git = "https://github.com/paritytech/wasmi.git"}
 wabt = "0.2.0"
-snowflake = "1.2"
\ No newline at end of file
+snowflake = "1.2"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = { version = "1.0", features = ["preserve_order"] }
```

Comment:
Wouldn't it be better to change the versions to just `"1"`?

Reply:
well wouldn't that unpin the version and allow the minor version to float?

Reply:
this was copied from elsewhere in the repo, i think this convo lives here: https://github.com/holochain/holochain-rust/issues/48